src/server.php

```diff
@@ -537,9 +537,10 @@ public function search($param) {
           } catch(Exception $e) {
             if($e->getMessage() == "UnableToConnectRedis") {
               VerboseJson::log(WARNING, 'UnableToConnectRedis in search');
-              $this->cache = new cache($this->config->get_value('cache_host', 'setup'),
-                $this->config->get_value('cache_port', 'setup'),
-                $this->config->get_value('cache_expire', 'setup'));
+              //$this->cache = new cache($this->config->get_value('cache_host', 'setup'),
+                //$this->config->get_value('cache_port', 'setup'),
+                //$this->config->get_value('cache_expire', 'setup'));
```

Comment:
Husk at fjerne den døde kode

Reply:
Jep, fjern udkommenteret kode. Der er versionshistorik, så hvis vi får brug for det, er det ikke forsvundet.